test/components/views/messages/MLocationBody-test.tsx

```diff
@@ -64,9 +65,11 @@ describe("MLocationBody", () => {
             });
             const component = getComponent();
 
-            // simulate error initialising map in maplibregl
-            // @ts-ignore
-            mockMap.emit("error", { status: 404 });
+            sleep(10).then(() => {
```

Comment:
why the sleep?

Reply:
Timings change in React 18 and otherwise this error was emitted too early and missed by the component being rendered